tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -104,3 +105,31 @@ Is RHODS Version Greater Or Equal Than
     ${comparison} =  GTE  ${ver}  ${target}
     [Return]  ${comparison}
 
+Move To Installed Operator instance
+    [Arguments]   ${operator_name}     ${tab_name}      ${namespace}
+    Switch To Administrator Perspective
+    Navigate to Installed Operators
+    Installed Operators Should Be Open
+    Log To Console   ${namespace}
+    Run Keyword If  "${namespace}" == "None"   Select Project By Name  All Projects
+    ...         ELSE   Select Project By Name   ${namespace}
+    Click On Searched Operator   ${operator_name}
+    Switch To New Tab       ${tab_name}
+    sleep    5
+
+Create tabname Instance For Installed Operator
```

Comment:
Is `tabname` supposed to be an expanded variable?

Reply:
Nope

Reply:
Then I would rename this keyword, because I don't understand what it's supposed to do

Reply:
hmm...this keyword is basically travel to the tabs present when we click on any operator and try to create and instance based on the tab name.

Reply:
I think one possible solution could be to something like:
1. rename the keyword `Move To Installed Operator instance`  to `Move To Installed Operator Page Tab`
2. rename the `Create tabname Instance For Installed Operator` to `Create Instance in Installed Operator Tab`

What do you guys think?

Reply:
Is the `tabname` the actual name of the browser tab that opens after clicking on an Operator? 
If so, I would avoid any reference to that. Maybe renaming to `Create Installed Operator Instance` would work? If it's supposed to be a general keyword to be used with any Operator, we don't really care about the `tabname` or the `operator` name, the keyword only knows that it's supposed to click on `Create`, and if it needs to be switched to a new tab before doing that it can be done either in the keyword or in an external keyword (like it is done now). 
`Switch To New Tab       ${tab_name}` isn't really necessary, as we know that a new tab will be opened and we can just switch to the newest one

Reply:
I think the tabname here isn't referred to browser tab but it refers to the Tab in the operator page. Like
![image](https://user-images.githubusercontent.com/88311595/149785713-0dad63ef-064e-478c-9f15-d5917d87b2fc.png)

Reply:
@lugi0 Comment mentioned by berto is right. One more thing i would like clarify is that "the keyword only knows that it's supposed to click on Create", it is just not Create in Operator we will have create+operator_tab_name

Reply:
Then I think I still do not understand what the keyword is doing. What is it creating exactly? And why does it need the name of one of those submenus?

Reply:
i am thinking to change the keyword names "Move To Installed Operator instance , Create tabname Instance For Installed Operator and Delete tabname Instance For Installed Operator" to "Move To Installed Operator Page Tab in Openshift,Create Notebook Instance For Installed Operator in Openshift and Delete Notebook Instance For Installed Operator in Openshift". Do you guys @lugi0 @bdattoma  think it will be still confusing or purposed name seems fine?

Reply:
@tarukumar I think that sounds good, I could not think of a better name for the keywords

Reply:
the problem I see is that `Create Notebook` is something very specific which might be valid only for Openvino